pkg/extension/grpc/generate_proto.sh

```
@@ -17,5 +17,6 @@ fi
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 protoc -I=$SCRIPT_DIR -I=$SCRIPT_DIR/$1 --go_out=$SCRIPT_DIR $SCRIPT_DIR/$1/common.proto
 protoc -I=$SCRIPT_DIR -I=$SCRIPT_DIR/$1 --go_out=$SCRIPT_DIR --go-grpc_out=$SCRIPT_DIR $SCRIPT_DIR/$1/kuadrant.proto
+protoc -I=$SCRIPT_DIR -I=$SCRIPT_DIR/$1 --go_out=$SCRIPT_DIR --go-grpc_out=$SCRIPT_DIR $SCRIPT_DIR/$1/descriptor_service.proto
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)